runtime/common/src/constants.rs

```diff
@@ -125,6 +137,11 @@ pub type MinReleaseTransfer = ConstU128<0>;
 pub const MAX_RELEASE_SCHEDULES: u32 = 50;
 // -end- TimeRelease Pallet ---
 
+#[cfg(any(feature = "frequency-testnet", feature = "frequency-local"))]
+// --- Timestamp Pallet ---
+pub type MinimumPeriod = ConstU64<0>;
+
+#[cfg(not(any(feature = "frequency-testnet", feature = "frequency-local")))]
```

Comment:
```suggestion
pub type MinimumPeriod = ConstU64<{prod_or_testnet_or_local!(SLOT_DURATION / 2, 0, 0)}>;
```
and remove the next line
`pub type MinimumPeriod = ConstU64<{ SLOT_DURATION / 2 }>;`

Reply:
Q: There is an `experimental` flag in [docs](https://wiki.polkadot.network/docs/maintain-guides-async-backing).  Is that obsolete?

Reply:
Yes. That flag has been removed.